src/hiero_sdk_python/query/account_balance_query.py

```
@@ -113,26 +148,28 @@ def execute(self, client) -> AccountBalance:
+
         Args:
             response: The full response from the network
-            
+
         Returns:
             The crypto get account balance response object
         """
         return response.cryptogetAccountBalance
-    
+
     def _is_payment_required(self):
```

Ruff (0.14.10)

Missing return type annotation for private function `_is_payment_required`

Add return type annotation: `bool`

(ANN202)